src/app/utils/__tests__/externalLinks.test.ts

```diff
@@ -0,0 +1,31 @@
+import nodeFetch from 'node-fetch'
+import * as externalLinksModule from '../externalLinks'
+
+jest.setTimeout(30_000)
+
+// Requesting external links is sometimes slow; and links will rarely break.
+// So only run this on CI, so local tests remain quick.
+const onlyRunOnCI = process.env.CI ? describe : describe.skip
+
+onlyRunOnCI('externalLinks', () => {
+  it('file should contain groups of links as objects', () => {
+    expect(Object.entries(externalLinksModule).length).toBeGreaterThan(0)
+    for (const [linksGroupName, linksGroup] of Object.entries(externalLinksModule)) {
+      expect(Object.entries(linksGroup).length).toBeGreaterThan(0)
+      for (const [linkName, url] of Object.entries(linksGroup)) {
+        expect(url).toMatch(/^https:/)
+      }
+    }
+  })
+
+  describe('should be reachable', () => {
```

Comment:
This should probably be `it`

Reply:
There are nested `it` calls inside. Would this be clearer:
```ts
  const externalLinksFlat = Object.entries(externalLinksModule).flatMap(([linksGroupName, linksGroup]) =>
    Object.entries(linksGroup).map(([linkName, url]) => ({ linksGroupName, linkName, url })),
  )
  describe('should be reachable', () => {
    it.concurrent.each(externalLinksFlat)('$linksGroupName $linkName $url', async ({ url }) => {
      const response = await nodeFetch(url, { method: 'HEAD' })
      expect(response.status).toBe(200)
    })
  })
```

Reply:
Missed the inner `it`, should be fine the existing way.